src/helper-methods.js

```diff
@@ -0,0 +1,67 @@
+import merge from "lodash/object/merge";
+import { Chart, Style } from "victory-util";
+import { Transform } from "react-art";
+
+
+module.exports = {
+  getStyles(props, defaultStyles) {
+    const style = props.style ? merge({}, defaultStyles, props.style) : defaultStyles;
+    return Style.removeInvisible(Chart.evaluateStyle(style));
+  },
+
+  getContentLength(props) {
+    if (props.children) {
+      const child = Chart.evaluateProp(props.children);
+      return `${child}`.split("\n").length;
+    }
+    return 1;
+  },
+
+  getDy(props, style) {
+    const length = this.getContentLength(props);
+    const lineHeight = 1.1; // used in art package
```

Comment:
Can this be extracted out into a higher scope? It's redefined every time this FN is called and it's kinda a magic number...